### PR TITLE
Fixes for health#provider (especially Python)

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: runtime/autoload/health/provider.vim [hunk 1]
```diff
@@ -70,7 +70,8 @@ function! s:system(cmd, ...) abort
   let jobid = jobstart(a:cmd, opts)
 
   if jobid < 1
-    call health#report_error(printf('Command error (job=%d): %s', jobid, s:shellify(a:cmd)))
+    call health#report_error(printf('Command error (job=%d): `%s` (in %s)',
+          \ jobid, s:shellify(a:cmd), string(getcwd())))
```
Comment (jamessan) [inline runtime/autoload/health/provider.vim]: `getcwd()` already returns a string, so the coercion shouldn't be necessary.

**Review by jamessan — COMMENTED** [review]
LGTM other than a small nit

File: runtime/autoload/health/provider.vim (hunk 1 above)
Comment (blueyed) [reply runtime/autoload/health/provider.vim]: It's meant to put it in quotes - but I'm happy to remove that.

Comment (jamessan) [reply runtime/autoload/health/provider.vim]: Ah, ok.  Then I'm fine either way.

**Review by justinmk — APPROVED** [review]

**Pull request merged**